### PR TITLE
Remove drupal_add_css() from book.module

### PR DESCRIPTION
**Pull request by adamtothemoon**
pr for: https://github.com/backdrop/backdrop-issues/issues/207

### CODE REVIEW

File: core/modules/book/book.module [hunk 1]
```diff
@@ -872,6 +872,11 @@ function book_node_view(Node $node, $view_mode) {
...
         '#markup' => theme('book_navigation', array('book_link' => $node->book)),
         '#weight' => 100,
+        '#attached' => array(
+          'css' => array(
+            drupal_get_path('module', 'book') . '/css/book.theme.css',
+          ),
+        ),
       );
     }
   }
```
Comment (jenlampton) [inline core/modules/book/book.module:882]: Looks like you forgot to actually remove the call to drupal_add_css()

```
-  drupal_add_css(drupal_get_path('module', 'book') . '/css/book.theme.css');
```

**Comment by jenlampton** [comment]
This PR needs work, so closing.  Please re-open when ready for a new review.

**Comment by jenlampton** [comment]
reopening.  Confirmed that there was no call to drupal_add_css in this issue.  Assigning to Quicksketch to review.

**Comment by quicksketch** [comment]
Looks like at some point the CSS was moved out of the .info file to use drupal_add_css(), then later switched to #attached. I filed a replacement PR at https://github.com/backdrop/backdrop/pull/333 that removes the line from the .info file as well.

**Pull request closed without merging**